src/bindings/c/include/openvino/c/ov_property.h

```diff
@@ -13,7 +13,7 @@
 
 #include "openvino/c/ov_common.h"
 
-typedef struct ov_property ov_property_t;
+typedef struct ov_property ov_properties_t;
```

Comment:
Why `ov_property` is `ov_properties_t`?

Reply:
ov_property is ov::AnyMap, which can contains multiple properties. Maybe change to ov_properties.

Reply:
But do we have a structure which provides ov::Any interface for C++?
Something like we discussed before:
```
struct {
void *
size
type
};
```

Reply:
Not yet, maybe there is very little chance to use such common data structure in C application. Let's hold it firstly?

Reply:
done